### ISSUE TITLE
Error report - Ubuntu 24.04 adding a bridge network with ip addresses set to off

### ISSUE THREAD

**Issue by jansyren** [canonical/lxd-ui#810]
Hi, every time I try to use the webUI to add an internal network without any pre configured ip ranges I get the error message below, as a workaround I have to create them with a network and then use the cli to remove the network configuration.

**Steps to reproduce:**

1. Install ubuntu desktop 24.04
2. Configure LXD with webUI
3. Enter webUI and configure network overriding default profile on ip settings and click off in the popup

snap    2.63+24.04
snapd   2.63+24.04
series  16
ubuntu  24.04
kernel  6.8.0-36-generic

lxd 5.21.1 LTS

I then receive the error pasted below:

```
Error: Minified React error #300; visit https://reactjs.org/docs/error-decoder.html?invariant=300 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at Zg (https://localhost:8443/ui/assets/index-b3f3eaed.js:38:19733)
    at Am (https://localhost:8443/ui/assets/index-b3f3eaed.js:40:3139)
    at B1 (https://localhost:8443/ui/assets/index-b3f3eaed.js:40:44801)
    at L1 (https://localhost:8443/ui/assets/index-b3f3eaed.js:40:39763)
    at m$ (https://localhost:8443/ui/assets/index-b3f3eaed.js:40:39691)
    at Cd (https://localhost:8443/ui/assets/index-b3f3eaed.js:40:39545)
    at Ym (https://localhost:8443/ui/assets/index-b3f3eaed.js:40:35913)
    at F0 (https://localhost:8443/ui/assets/index-b3f3eaed.js:40:36716)
    at hi (https://localhost:8443/ui/assets/index-b3f3eaed.js:38:3274)
    at eo (https://localhost:8443/ui/assets/index-b3f3eaed.js:40:37163)
```

**Comment by edlerd** [comment]
Thank you for the detailed report.

While the underlying issue has been fixed with #792, it is not yet released to the 5.21/stable channel. For the time being, you can rely on the 5.21/edge channel, which already includes the fix. Though, I wouldn't suggest running the edge channel in production. I expect the fix to be also available via the 5.21/candidate channel shortly.

**Comment by jansyren** [comment]
Thanks, I will live with my workaround until it is out in stable version. Thanks for the swift reply.

**Comment by edlerd** [comment]
Closing this one as the fix is now available in 5.21/candidate channel. Though it might take a bit more time until a new 5.21/stable is released.
